security/fma-incident-response-improvements.md

```diff
@@ -0,0 +1,254 @@
+  - [Portal](https://github.com/ethereum-optimism/specs/blob/ab35e8181f34614503a03e18a282cce69ea634fe/specs/fault-proof/stage-one/optimism-portal.md)
+  - [AnchorStateRegistry](https://github.com/ethereum-optimism/specs/blob/7706b68c6f8f4172f9396c03175e6c8cb299bfbc/specs/fault-proof/stage-one/anchor-state-registry.md)
+  - [DelayedWETH](https://github.com/ethereum-optimism/specs/blob/a0c94920a3c3b6b3527e4f9c3f07b787cd9fe2ad/specs/fault-proof/stage-one/bond-incentives.md#delayedweth)
+
+## FM1: Contract deployment and/or initialization failures
+
+### Description
+
+Any of the modified contracts could be deployed or initialized incorrectly leading to a variety of failure modes, depending on the exact contract impacted.
+
+### Risk Assessment
+
+- Impact: HIGH/CRITICAL
+  - Reasoning: Highly depends on the exact failure mode but can be critical in the worst case.
+- Likelihood: LOW
+  - Reasoning: Very likely to be covered by both OPCM, testing, and auditing.
+
+### Mitigations
+
+- Validation via OPCM
+- Integration testing
+- 3rd party audit
+- Testnet deployment
+
+### Detection
+
+Detection for this failure mode is highly dependent on the exact manner in which the mode occurs. Given the complexity in detecting this failure mode and the overlap with existing OPCM validation, we do not propose any new detection mechanisms for this mode.
+
+### Recovery Path(s)
+
+- Contract redeployment if necessary
+- Superchain-wide pause mechanism
+
+## FM2: Legacy runbook/response confusion
+
+### Description
+
+Operators or Guardians familiar with older versions of these contracts may follow outdated procedures during an incident.
+
+For example:
+
+- Calling only `DelayedWETH.hold()` without subsequent `DelayedWETH.transferFrom()` on a legacy `DelayedWETH`
+- Failing to call `OptimismPortal.setRespectedGameType(type(uint32).max)` to invalidate games
+- Following outdated emergency procedures / misunderstanding version-specific behaviors
+
+This risk is heightened during emergency situations when quick action is required.
+
+### Risk Assessment
+
+- Impact: HIGH/CRITICAL
+  - Reasoning: Exact risk depends on the specific incident response mechanism impacted. For instance, if the chain operator calls `OptimismPortal.setRespectedGameType` assuming that it will invalidate all existing games, the operator may not understand that existing games can _still_ be used to finalize withdrawals (critical risk). Other failure modes within this category are generally MEDIUM/HIGH.
+- Likelihood: LOW
+  - Reasoning: Only applies during the first ~2 weeks after the upgrade when existing `FaultDisputeGame` contracts may still refer to the legacy `DelayedWETH` contract. Failure to hold funds would be easily noticable, only results in any impact if the `hold` operation occurs very close to the finalization time. Does not apply to any future upgrades.
+
+### Mitigations
+
+- Clear version documentation
```

Comment:
Do we have this documentation or is that part of the "version-aware runbooks" action item?

Reply:
It's part of the runbook already